internal/controllers/httpsedge_controller.go

```diff
@@ -315,6 +319,40 @@ func (r *HTTPSEdgeReconciler) setEdgeRouteResponseHeaders(ctx context.Context, e
 	return err
 }
 
+func (r *HTTPSEdgeReconciler) setEdgeRouteWebhookVerification(ctx context.Context, edge *ingressv1alpha1.HTTPSEdge, routeID string, webhookValidation *ingressv1alpha1.EndpointWebhookValidation) error {
+	client := r.NgrokClientset.EdgeModules().HTTPS().Routes().WebhookVerification()
+	if webhookValidation == nil {
+		return client.Delete(ctx, &ngrok.EdgeRouteItem{EdgeID: edge.Status.ID, ID: routeID})
+	}
+
+	// TODO: Get the secret from the store
+	secret := &v1.Secret{}
+	err := r.Client.Get(ctx, types.NamespacedName{
+		Name:      webhookValidation.SecretRef.Name,
+		Namespace: edge.Namespace,
+	}, secret)
+	if err != nil {
+		return err
```

Comment:
so does this make the event the user see's the plain k8s get failure of "resource not found". At some point I think we'll want to review the wording and error messages for common configuration error cases I think

Reply:
Yes, that should be the error returned. As an aside, I was also wondering if its worth it to upgrade our logging at some point to be able to do debug logging. I think we are getting close to the point where having info log for everything is too noisy, but for development or troubleshooting we'd want those debug logs.

Reply:
yeah, I think that will be a good idea. I think once things stabalize we can review all the events/logs that happen in some flows and tune things from there